code/modules/research/designs/weapon_designs.dm

```diff
@@ -126,7 +126,7 @@
 	category = list("Weapons")
 
 /datum/design/smg
-	name = "Prototype Submachine Gun"
+	name = "NanoTrasen Saber SMG"
```

Comment:
Does this actually need "NanoTrasen" at the beginning? Seems kind of redundant to me but I don't know anything. And either way it should be \improper